src/search/pdbs/cegar.cc

```diff
@@ -225,10 +226,8 @@ bool CEGAR::time_limit_reached(
 
 unique_ptr<PatternInfo> CEGAR::compute_pattern_info(Pattern &&pattern) const {
     vector<int> op_cost;
-    bool compute_plan = true;
-    shared_ptr<PatternDatabase> pdb =
-        make_shared<PatternDatabase>(task_proxy, pattern, op_cost, compute_plan, rng, use_wildcard_plans);
-    vector<vector<OperatorID>> plan = pdb->extract_wildcard_plan();
+    auto [pdb, plan] = compute_pdb_and_wildcard_plan(
+        task_proxy, pattern, op_cost, rng, use_wildcard_plans);
```

Comment:
I'm still a bit confused with the naming. What does `compute_..._wildcard_plan` do with `use_wildcard_plans=false`?

Reply:
I would like to change it back to `compute_pdb_and_plan` (which you didn't like initially) because I think that this is indeed confusing. If `use_wildcard_plans=false`, the method computes a regular plan, though still using the same data structure that you seem to dislike so much, by having the inner vector containing exactly one operator ID for each plan step.

Reply:
Fine with me. What I dislike is more the place where this code lives. I think this is a very specific computation that is useful for one particular use case and implemented in a general interface. But I'm fine with leaving it as is for this issue.